influxdb-backup/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM influxdb:1.2.4
+
+# Backup the following databases, seperator ":"
+ENV DATABASES=demo:_internal:testdb
+
+#default InfluxDB host
+ENV INFLUX_HOST=influxdb
+
+# Change workdir
+WORKDIR "/tmp"
+
+# Backup script
+COPY backup.sh /bin/backup.sh
+RUN chmod +x /bin/backup.sh
+
+# Backup directory
+RUN mkdir -p /tmp/backup
```

Comment:
since this is external, this should not be in /tmp. `/var/lib/influxdb-backup` (similar to `/var/lib/grafana`) is the usual convention.